types/django-bananas/themes.d.ts

```diff
@@ -0,0 +1,11 @@
+export function applyThemeDefaults(theme: any): any;
+export function createBananasTheme(theme: any): any;
+export function extendTheme(theme: any, overrides: any): any;
```

Comment:
the any here is too vague. The `theme` seems to extends an object, at least should be say keyed interface, or a rector or just T extends object. It can be also:
```ts
interface Theme {
    [key: string]: string | Theme;
}
```